src/main/java/de/zalando/aruha/nakadi/domain/Subscription.java

```diff
@@ -0,0 +1,88 @@
+package de.zalando.aruha.nakadi.domain;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Collections;
+import java.util.Set;
+
+import static com.google.common.collect.Sets.newTreeSet;
+
+public class Subscription {
+
+    @Nullable
+    private String id;
+
+    @NotNull
+    private String owningApplication;
+
+    @NotNull
+    @Size(min = 1, max = 1)
+    private Set<String> eventTypes;
+
+    private String useCase = "none";
+
+    private DateTime createdAt = new DateTime(DateTimeZone.UTC);
```

Comment:
@rcillo As it is a field inside json it's kinda difficult to populate it with some default value on DB level :) So I decided to go with easiest path.

@antban I think we even defined in our API that this field should not be set when creating subscriptions. If we let user set it - it will not be the creation time then.